cloud-service-provider/aws/eks/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "eu-west-1"
+}
+
+variable "cluster_name" {
+  description = "EKS cluster name"
+  type        = string
+  default     = null
```

Comment:
A suggestion, remove the default so that an input is required and it prompts the user if missing.

Reply:
should the region also not have a default. Alternately a default region establishes that we can find the instance type mentioned in that region.